src/vm-repair/azext_vm_repair/_validators.py

```diff
@@ -45,7 +45,7 @@ def validate_create(cmd, namespace):
         namespace.repair_vm_name = ('repair-' + namespace.vm_name)[:14] + '_'
 
     # Check copy disk name
-    timestamp = datetime.utcnow().strftime('%Y%m%d%H%M%S.%f')
+    timestamp = datetime.utcnow().strftime('%Y%m%d%H%M%S')
```

Comment:
Please write the issue you fixed in history notes

Reply:
added history, thanks.